test/testAbi.ts

```diff
@@ -1,16 +1,17 @@
 require('it-each')({ testPerIteration: true });
-const _ = require('lodash');
-const constants = require('./../src/constants');
-const abi = require('./../src/ethereumAbi');
-const randomWords = require('random-words');
-const crypto = require('crypto');
-const ethersAbi = require('@ethersproject/abi');
-const expect = require('chai').expect;
-const helpers = require('./testUtil/helpers');
-const question = require('readline-sync').question;
-const seedrandom = require('seedrandom');
-
-const encoder = new ethersAbi.AbiCoder;
+import { AbiCoder } from '@ethersproject/abi';
+import { expect as expect } from 'chai';
+import crypto from 'crypto';
+import _ from 'lodash';
+import randomWords from 'random-words';
+import { question as question } from 'readline-sync';
+import seedrandom from 'seedrandom';
+import { ETH_ABI_LATTICE_FW_TYPE_MAP, getFwVersionConst, HARDENED_OFFSET } from '../src/constants';
+import abi from './../src/ethereumAbi';
+import funcDef from './testUtil/etherscanABI_0xV2.json';
+import helpers from './testUtil/helpers';
+
+const encoder = new AbiCoder();
```

Comment:
Are the parens needed here?

Reply:
@douglance bump on this one

Reply:
Yeah, basically instead of importing `ethers` and using the `AbiCoder` off of it, I'm importing the `AbiCoder` directly. 

This should behave the same way as before.